### ISSUE TITLE
Default mail app gone?

### ISSUE THREAD

**Issue by littleblack111** [Alex313031/thorium#735]
**System Details**
 - OS: [e.g.  Ubuntu 22.04, Windows 7, 8.1, 10, 11]
 - Thorium Version [e.g.107.0.5347.0]
 - OS: Arch Linux
 - Thorium Version: 124.0.6367.218

**Problem**
on mail PWAs like shortwave. make default mail app does not show up(popup)

**Screenshots**

**Additional Notes**
Stock Chromium works. Firefox works

**Comment by gz83** [comment]
Please try version 150 if you are still using thorium.

**Comment by littleblack111** [comment]
Nope, I moved on to other browsers, so I’ll close it feel free to reopen if the bug still persist
